fuzz/client.c

```diff
@@ -25,6 +26,23 @@ extern int rand_predictable;
 /* unused, to avoid warning. */
 static int idx;
 
+#define FUZZTIME 1485898104
+
+#ifdef _WIN32
+# ifdef _TIME64_T_DEFINED
+time64_t _time64(time64_t *t)
+# else
+time_t _time(time_t *t)
```

Comment:
In other words choice is between
```
#define _INC_TIME_INL
#include <time.h>
...
#if defined(_WIN32) && defined(_TIME64_T_DEFINED)
time64_t _time64(time64_t *t) { implementation }
#endif
time_t time(time_t *t) { implementation }
```
and
```
#include <time.h>
...
#if defined(_WIN32) && defined(_TIME64_T_DEFINED)
time64_t _time64(time64_t *t) { implementation }
#endif
#if !defined(_WIN32) || !defined(_MSC_VER)
time_t time(time_t *t) { implementation }
#endif
```
Even though former is actually better *today*, my vote goes for the latter [because it appears more robust in long run]. But feel free to pick either.